app/src/main/java/org/simple/clinic/teleconsultlog/prescription/doctorinfo/TeleconsultDoctorInfoView.kt

```diff
@@ -67,6 +67,12 @@ class TeleconsultDoctorInfoView(
     )
   }
 
+  val medicalInstructions: String
+    get() = delegate.currentModel.instructions.orEmpty()
+
+  val medicalRegistrationId: String
+    get() = delegate.currentModel.medicalRegistrationId.orEmpty()
```

Comment:
The `currentModel` property on `MobiusDelegate` is deprecated since we added it only for the purposes of Mobius migration. Since we are done with that, I was thinking we can remove this property.

If we decide to keep it for cases like these, however, we might want to remove the deprecation notice.

Reply:
Yep, I forgot to ask you about removing deprecation notice.

Reply:
Let's remove it then.